Init analysis c2r role (#2)
oamg/infra.convert2rhel: README.md

# Ansible Convert2RHEL Collection

> [![CI](https://github.com/redhat-cop/infra.convert2rhel/workflows/CI/badge.svg?event=push)](https://github.com/redhat-cop/infra.convert2rhel/actions) [![Lint](https://github.com/redhat-cop/infra.convert2rhel/workflows/Yaml%20and%20Ansible%20Lint/badge.svg?event=push)](https://github.com/redhat-cop/infra.convert2rhel/actions) [![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7438/badge)](https://bestpractices.coreinfrastructure.org/projects/7438)

Reviewer: ```suggestion
[![CI](https://github.com/oamg/infra.convert2rhel/workflows/CI/badge.svg?event=push)](https://github.com/oamg/infra.convert2rhel/actions) [![Lint](https://github.com/oamg/infra.convert2rhel/workflows/Yaml%20and%20Ansible%20Lint/badge.svg?event=push)](https://github.com/oamg/infra.convert2rhel/actions) [![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7438/badge)](https://bestpractices.coreinfrastructure.org/projects/7438)
```

> <!-- [![Codecov](https://img.shields.io/codecov/c/github/redhat-cop/infra.convert2rhel)](https://codecov.io/gh/redhat-cop/infra.convert2rhel) -->

Reviewer: ```suggestion
<!-- [![Codecov](https://img.shields.io/codecov/c/github/oamg/infra.convert2rhel)](https://codecov.io/gh/oamg/infra.convert2rhel) -->
```

## Reporting issues

> Please open a [new issue](https://github.com/redhat-cop/infra.convert2rhel/issues/new/choose) for any bugs or security vulnerabilities you may encounter. We also invite you to open an issue if you have ideas on how we can improve the solution or want to make a suggestion for enhancment.

Reviewer: ```suggestion
Please open a [new issue](https://github.com/oamg/infra.convert2rhel/issues/new/choose) for any bugs or security vulnerabilities you may encounter. We also invite you to open an issue if you have ideas on how we can improve the solution or want to make a suggestion for enhancment.
```

## More information

> This Ansible collection is just one building block of our larger initiative to make Convert2RHEL conversion automation that works at enterprise scale. Learn more about our end-to-end approach for automating conversions in these [blog posts](https://www.redhat.com/en/blog?search=convert2rhel).

Author: We could probably add more, maybe https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux/centos-migration

Author: can be done at the end ..

## Release notes

> See the [changelog](https://github.com/redhat-cop/infra.convert2rhel/tree/main/CHANGELOG.rst).

Reviewer: ```suggestion
See the [changelog](https://github.com/oamg/infra.convert2rhel/tree/main/CHANGELOG.rst).
```
